src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleList } from '../components/PeopleList/PeopleList';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
```

Comment:
```suggestion
  const [peoples, setPeoples] = useState<Person[]>([]);
```

will be more informative